Header  header
int32   field1 #17
float32 field2 #58.2
string  field3 #"aldfkja"
